### PR TITLE
Fix l10n_ca_account_check_writing

### PR DESCRIPTION
**Pull request by ghost**
The field supplier_invoice_number is already implemented in the module account_voucher_supplier_invoice_number

### CODE REVIEW

**Comment by ghost** [comment]
@joaoalf  I will need your review here

File: .travis.yml [hunk 1]
```diff
@@ -33,6 +33,7 @@ install:
   - travis_install_nightly
   - pip install num2words
   - git clone --depth=1 https://github.com/OCA/account-fiscal-rules.git /tmp/fiscal-rules -b ${VERSION}
+  - git clone --depth=1 https://github.com/OCA/account-payment.git /tmp/account-payment -b ${VERSION}
```
Comment (ventilooo) [inline .travis.yml:36]: keep the ident, the `-b ${VERSION}` should be right under the upper one.

Comment (ghost) [inline .travis.yml:36]: I prefer no indent at all, over indents that may change over time.

**Comment by Ehtaga** [comment]
Good! :+1:

**Pull request merged**